lib/ReactViews/Workbench/Controls/ViewingControls.jsx

```diff
@@ -403,7 +407,7 @@ const ViewingControls = observer(
       const classList = {
         [Styles.noZoom]: !canZoom,
         [Styles.noSplit]: !canSplit,
-        [Styles.noInfo]: !item.showsInfo
```

Comment:
```js
const canSplit =
        !item.terria.configParameters.disableSplitter &&
        item.supportsSplitting &&
        defined(item.splitDirection) &&
        item.terria.currentViewer.canShowSplitter;```

We should change the `supportsSplitting` to `disableSplitter` or remove the entire `classList`. I don't think it has any effect any more since the classes are not defined in the `.scss` style file.

Reply:
Nice find, I've removed all SCSS